### DISCUSSION TITLE
Adding Mojo::JSON::Pointer to ojo

### DISCUSSION THREAD

**Discussion by s1037989** [mojolicious/mojo#1851] [Ideas]
Sometimes it's useful to point to a specific JSON value using a pointer:
```
$ perl -Mojo -E 'say Mojo::JSON::Pointer->new(j(<STDIN>))->get(@ARGV)' /hello < hello.json
Hello Mojo!
```

Could we add Mojo::JSON::Pointer to "ojo" to help out a little bit?
```
$ perl -Mojo -E 'say J->get(@ARGV)' /hello < hello.json
Hello Mojo!
```
- Note: Obviously this `J` function doesn't exist.  Default to `<STDIN>` unless a JSON string or Mojo::File object is passed.

**Comment by jberger** [comment]
An interesting idea. Usually I just pipe the json out to `jq` but I imagine there are circumstances where you want the result back in your one-liner.

**Comment by s1037989** [comment]
That's right, I very often want the result in my one-liner, and sometimes
I'll pipe to `jq` and pipe again to another perl or ojo. It's fine, of
course, but every time I ask myself, "could this be better?"

Mojolicious can do so much heavy lifting, and the `get` command can point
to my json, so I'm often reaching for a Mojolicious way to do something
similar but with a resource that isn't a URL.
